Content.Shared/DrawDepth/DrawDepth.cs

```diff
@@ -9,33 +9,35 @@ public enum DrawDepth
         /// <summary>
         ///     This is for sub-floors, the floors you see after prying off a tile.
         /// </summary>
-        LowFloors = DrawDepthTag.Default - 11,
+        LowFloors = DrawDepthTag.Default - 16,
 
         // various entity types that require different
         // draw depths, as to avoid hiding
         #region SubfloorEntities
-        ThickPipe = DrawDepthTag.Default - 10,
-        ThickWire = DrawDepthTag.Default - 9,
-        ThinPipe = DrawDepthTag.Default - 8,
-        ThinWire = DrawDepthTag.Default - 7,
+        ThickPipe = DrawDepthTag.Default - 15,
+        ThickWire = DrawDepthTag.Default - 14,
+        ThinPipe = DrawDepthTag.Default - 13,
+        ThinWire = DrawDepthTag.Default - 12,
         #endregion
 
         /// <summary>
         ///     Things that are beneath regular floors.
         /// </summary>
-        BelowFloor = DrawDepthTag.Default - 7,
+        BelowFloor = DrawDepthTag.Default - 11,
 
         /// <summary>
         ///     Used for entities like carpets.
         /// </summary>
-        FloorTiles = DrawDepthTag.Default - 6,
+        FloorTiles = DrawDepthTag.Default - 10,
 
         /// <summary>
         ///     Things that are actually right on the floor, like puddles. This does not mean objects like
         ///     tables, even though they are technically "on the floor".
         /// </summary>
-        FloorObjects = DrawDepthTag.Default - 5,
+        FloorObjects = DrawDepthTag.Default - 9,
 
```

Comment:
Why the gap?

Reply:
Because I changed it so it adds a number to subfloor entities' draw depth instead of setting it to floor objects + 1. this retains their relative draw depth. Without the gap, the new draw depth would land in small mobs, floor objects etc., making subfloors appear above rodents for example. With the gap, the new draw depth doesn't clash with anything. So now subfloors are -15 to -12, as of now the difference added is 7, and when using a t-ray scanner it lands in -8 to -5. Without the gap it would be -4 to -1, clashing with  existing draw depths.

To my knowledge there's no method for more fine-grained draw depth control.